src/styles/page-template/index.md.njk

```diff
@@ -32,12 +32,14 @@ This example shows the minimum required for a GOV.UK page.
 
 ## Customised page template
 
-You can customise the page template, for example, by adding a service name and navigation or adding a block before the main container to include a [back link](/components/back-link/) or [phase banner](/components/phase-banner/).
+You can customise the page template, for example, by adding a service name and navigation or adding a block before the main container to include a a [back link](/components/back-link/) or [phase banner](/components/phase-banner/).
```

Comment:
This is the current wording in the guidance. The long sentence confused me...

Are 'adding a service name and navigation' and 'adding a block before the main container to include a back link or phase banner' two separate things?

Or is it the case that, to include a back link or phase banner, you can EITHER add a service name before the main container OR add a block before the main container?

Reply:
Delete second 'a'.

Reply:
```suggestion
You can customise the page template, for example, by:

- adding a service name and navigation
- including a [back link](/components/back-link/) or [phase banner](/components/phase-banner/)

```